pkgs/development/libraries/fftw/default.nix

```diff
@@ -85,5 +85,7 @@ stdenv.mkDerivation (finalAttrs: {
       }.${precision}
     ];
     platforms = platforms.unix;
+    # quad-precision requires libquadmath from gfortran, but libquadmath is not supported on aarch64
+    badPlatforms = if (precision == "quad-precision") then platforms.aarch64 else [];
```

Comment:
```suggestion
    badPlatforms = lib.optional (precision == "quad-precision") platforms.aarch64;
```

Reply:
Thanks, somehow missed that entry in the "Code Conventions" part of CONTRIBUTING.md!

Committed, squashed, added you as Co-author.

Reply:
This suggestion caused the package to try and build on aarch64. Fixed by using `lib.optionals` instead, since `platforms.aarch64` is already a list. 

Looks like an instance of this: https://discourse.nixos.org/t/lib-optional-considered-harmful/7081